python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1302,6 +1305,10 @@ async def _wait_hard_point_test_ok(self, hp: int) -> None:
             else:
                 self.log.info(f"Hard point {hp} test state: {hp_test_state!r}.")
 
+            await self.rem.mtm1m3.evt_heartbeat.next(
```

Comment:
you might want to wrap this await here with a `try/except` and catch and treat timeout errors. It might be hard for users to understand why there is a heartbeat timeout coming from this loop. Consider something like:

```
try:
    await self.rem.mtm1m3.evt_heartbeat.next(...)
expect asyncio.TimeoutError:
    raise RuntimeError(f"No heartbeat received from M1M3 in the last {self.timeout_hardpoint_test_status}s while waiting for hard point data information. Check CSC liveliness.")
```